mbed/mbed.py

```diff
@@ -1389,6 +1398,8 @@ def update_tools(self, path):
             with cd(os.path.join(path, tools_dir)):
                 try:
                     action("Updating the mbed 2.0 SDK tools...")
+                    print path
+                    print os.path.join(path, tools_dir)
```

Comment:
These look like debug prints. If they are, could you remove them?

Reply:
Thanks for spotting this.